SDK/articles/AddingCustomContextMenu.md

```diff
@@ -0,0 +1,248 @@
+# Adding custom context menu to provider folder node
+
```

Comment:
Can you add a note that this functionality is introduced with an upcoming update to VS 2017?